### PR TITLE
Daily reports class

### PR DESCRIPTION
**Pull request by mSitkovets**
### Related Issue:

None

### Proposed Changes

Adding a class named `DailyReports` to wrap the utility functions defined in `get_data.py`. The `CovidAPIv2Integrator` would have a `has_a` relationship with `DailyReports` and have it passed into its initializer so that it can be used to get the overall daily reports for all countries or specifically for the US by sending the `US=True` flag when calling the `get_data_daily_reports` method. In addition, this could further be refactored in the future to instead pass in the name of the country as an argument if we wish to add this functionality to the Covid19 API.  

### Checklist

* [x] Tests
* [ ] Translations
* [ ] Documentation

### CODE REVIEW

**Review by nat236919 — APPROVED** [review]
Thank you for the PR. Even though there are some errors which I will fix accordingly, it's still LGTM.

I will also leave some comments on what was missing

File: app/integrators/covid_api_v2_integrator.py [hunk 1]
```diff
@@ -24,8 +24,7 @@
                                          TimeseriesUSDataModel,
                                          TimeseriesUSInfoModel,
                                          TimeseriesUSModel, TotalModel)
-from utils.get_data import (get_data_daily_reports,
-                              get_data_daily_reports_us, get_data_lookup_table,
+from utils.get_data import (DailyReports, get_data_lookup_table,
```
Comment (nat236919) [inline app/integrators/covid_api_v2_integrator.py:27]: **get_country** also needs to change to

`self.df = self.daily_reports.get_data_daily_reports() # Get base data`

File: app/utils/get_data.py [hunk 2]
```diff
@@ -27,38 +27,23 @@ def get_data_lookup_table() -> Dict[str, str]:
...
-    df = helper_df_cols_cleaning(df, concerned_columns, int)
-    
-    return df
-
+class DailyReports:
+    def __init__(self) -> None: 
+        self.latest_base_url = helper_get_latest_data_url(JHU_CSSE_FILE_PATHS['BASE_URL_DAILY_REPORTS'])
+
+    # Get data from daily reports
+    def get_data_daily_reports(self, US=False) -> pd.DataFrame:
```
Comment (nat236919) [inline app/utils/get_data.py:35]: A type hint is missing from **US**

**Current**
`get_data_daily_reports(self, US=False)`

**Ideal**
`get_data_daily_reports(self, US: bool = False)`

File: app/utils/get_data.py [hunk 3]
```diff
@@ -27,38 +27,23 @@ def get_data_lookup_table() -> Dict[str, str]:
...
+        """ Get data from BASE_URL_DAILY_REPORTS """
+        # Extract the data
+        if US is False:
+            df = pd.read_csv(self.latest_base_url)
+        else: 
+             df = pd.read_csv(JHU_CSSE_FILE_PATHS['BASE_URL_DAILY_REPORTS_US'])
```
Comment (nat236919) [inline app/utils/get_data.py:41]: **helper_get_latest_data_url** is required to get data from URL.

**Comment by nat236919** [comment]
@all-contributors please add @mSitkovets for code

**Comment by mSitkovets** [comment]
@nat236919 should I make these changes on a new PR since this one has been merged?

**Comment by nat236919** [comment]
@mSitkovets Nope, that won't be necessary. I've already patched it with https://github.com/nat236919/covid19-api/commit/be20ce7e877b965da403221b5cb7f21f80209798

**Pull request merged**